### ISSUE TITLE
Sequence navigation should use links with slugs

### ISSUE THREAD

**Issue by zackmdavis** [LessWrong2/Lesswrong2#3091]
The little **<** and **>** Sequence navigation buttons at the top of a post that is part of a Sequence link to the next and previous post using ugly, incomprehensible URLs like https://www.lesswrong.com/s/Kqs6GR7F5xziuSyGZ/p/7FSwbFpDsca7uXpQ2, but I argue they should use the form of URL with the pretty human-readable ["slug"](https://en.wikipedia.org/wiki/Clean_URL#Slug), like https://www.lesswrong.com/posts/7FSwbFpDsca7uXpQ2/quantum-explanations.

**Comment by Discordius** [comment]
Yeah, I agree with that. The problem is that those are special links that link to a post from the context of a sequence. I.e. sometimes you have a post that is in multiple sequences, so you have to be able to link to a post in the context of some other sequence. 

So, I think the way we should structure the links is to have the sequence as a URL parameter, and the the URL's look like this: 

https://www.lesswrong.com/posts/7FSwbFpDsca7uXpQ2/quantum-explanations?sequenceId=Kqs6GR7F5xziuSyGZ
